src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,283 @@
+} from "../../app/features/mondomarket";
+
+import { AddMondoItem, MondoDetailsViewer } from "../../services/mondomarket";
+import Modal from "../../components/modal/Modal";
+
+const MondoMarket = () => {
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const [addModal, setAddModal] = useState(false);
+  const [mondoModal, setMondoModal] = useState(false);
+  const [reload, setReload] = useState(false);
+  const [isTop, setIsTop] = useState(false);
```

ESLint

[error] 'isTop' is assigned a value but never used.

(no-unused-vars)